unstructured/partition/text.py

```diff
@@ -261,11 +261,13 @@ def element_from_text(
     coordinate_system: Optional[CoordinateSystem] = None,
 ) -> Element:
     if is_bulleted_text(text):
-        return ListItem(
-            text=clean_bullets(text),
-            coordinates=coordinates,
-            coordinate_system=coordinate_system,
-        )
+        clean_text = clean_bullets(text)
+        if clean_text:
+            return ListItem(
+                text=clean_text,
+                coordinates=coordinates,
+                coordinate_system=coordinate_system,
+            )
```

Comment:
If the `if` statement fails, there's no fallback, so this function returns `None`.